crates/zir-codegen/tests/backend_agnostic.rs

```diff
@@ -43,7 +43,8 @@ fn test_compile_const_function() {
     let body = create_const_function(&arena, 42);
 
     let mut backend = create_backend(CodegenConfig::default());
```

Comment:
Avoid imperative `create_backend`, we must declare `mir` tests once, but execute for each `CodegenBackend`